src/actions/index.js

```diff
@@ -89,7 +89,6 @@ export {
   resetPassword,
   updatePassword,
   updateUser,
-  showAllUsers,
```

Comment:
@nileshgulia1 this was introduced by
https://github.com/plone/volto/pull/1733

I think we are removing it with this changes... Can you take a look?